assets/js/utils/rtcUtils.ts

```diff
@@ -27,34 +31,45 @@ export const mergedScreenRef: MergedScreenRef = {
 
 const sourceIds: { audio: string; video: string } = { audio: "", video: "" };
 
-export const findTrackByType = (client: Client, sourceType: SourceType) => {
-  return presenterStreams[client.email].getTracks().find((elem) => elem.kind == sourceType);
+export const findTrackByType = (client: Client | Presenter, sourceType: SourceType) => {
```

Comment:
We probably should create an interface for the common part of `Client` and `Presenter`.

Reply:
Done